src/js/components/Organization/OpinionsIgnoredList.jsx

```diff
@@ -38,7 +38,15 @@ export default class OpinionsIgnoredList extends Component {
     return (
       <div className="guidelist card-child__list-group">
         {this.state.organizationsIgnored.map((oneOrganization) => (
-          <VoterGuideDisplayForList key={oneOrganization.organization_we_vote_id} {...oneOrganization}>
+          <VoterGuideDisplayForList
```

Comment:
Hi @gautam-ergo I notice that you changed the prop names to camelCase (which is good), but I'm not seeing the props in src/js/components/VoterGuide/VoterGuideDisplayForList.jsx that you updated the incoming props values to match camelCase:
```
    if (this.props.organization_we_vote_id === undefined) {
      // console.log('VoterGuideDisplayForList this.props.organization_we_vote_id === undefined');
      return null;
    }

    // We package up the above variables to mimic a position
    const position = this.props;

    const {
      organization_we_vote_id: organizationWeVoteId,
      voter_guide_image_url_large: voterGuideImageUrlLarge,
    } = this.props; // twitter_followers_count,

```